documentation/source/users/rmg/input.rst

```diff
@@ -481,6 +481,35 @@ For example ::
 		toleranceMoveSurfaceReactionToCore=5.0,
 	)
 
+Advanced Setting:  Dynamics Criterion Weights
+---------------------------------------------
+In reaction networks reactions that change the number of radicals tend to be much more important
+than other reactions.  Also in many cases specific species within a model are of interest.  
+
+In these sorts of situations RMG implements two different types of weights:
```

Comment:
Add an extra line after this so that `Reaction Weights` appears on a separate line in the html.